shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm

```diff
@@ -41,20 +41,6 @@
 
 void VsyncWaiterIOS::AwaitVSync() {
   double new_max_refresh_rate = [DisplayLinkManager displayRefreshRate];
-  if (fml::TaskRunnerChecker::RunsOnTheSameThread(
-          task_runners_.GetRasterTaskRunner()->GetTaskQueueId(),
-          task_runners_.GetPlatformTaskRunner()->GetTaskQueueId())) {
-    BOOL isRunningOnMac = NO;
-    if (@available(iOS 14.0, *)) {
-      isRunningOnMac = [NSProcessInfo processInfo].iOSAppOnMac;
-    }
-    if (!isRunningOnMac) {
-      // Pressure tested on iPhone 13 pro, the oldest iPhone that supports refresh rate greater than
-      // 60fps. A flutter app can handle fast scrolling on 80 fps with 6 PlatformViews in the scene
-      // at the same time.
-      new_max_refresh_rate = 80;
```

Comment:
Ask to learn: Does this mean that we wanted to cap to 80FPS when there's platform view on screen (but if no platform view, it's capped at 120FPS)? By removing this, we remove the 80FPS cap, and let OS to decide what refresh rate is.

Reply:
When there were 5 or more platform views, we set the cap to 80. Now we always set to whatever the maximum rate is, in this case 120. AFAIK, we don't fully support adjusting the frame rate.

Reply:
For background, check out https://github.com/flutter/flutter/issues/116640 https://github.com/flutter/engine/pull/39172 (and https://github.com/flutter/engine/pull/39172), there's a lot of info in that issue.